sky/unit/test/gestures/scale_test.dart

```diff
@@ -36,7 +36,7 @@ void main() {
 
     TestPointer pointer1 = new TestPointer(1);
 
-    ui.PointerEvent down = pointer1.down(new Point(10.0, 10.0));
+    PointerInputEvent down = pointer1.down(new ui.Point(10.0, 10.0));
```

Comment:
why do you need to say ui.Point? Did this test not work before? You should be able to just remove the dart:ui import entirely, no?